serialization/json-ld.md

```diff
@@ -0,0 +1,41 @@
+# JSON-LD
+JSON-LD is a JSON-based format to encode RDF graphs. Its documentation can be found [here](https://www.w3.org/TR/json-ld11/).
+
+The JSON-LD is an RDF format and follows the serialization rules of the [rdf SPDX serialization specification](rdf.md).
+
+## XCollection
+
+The following XCollection properties are mapped to JSON-LD syntax specifications.
+Any properties not listed below should be serialized as part of the XCollection element itself within
+the JSON-LD serialized data.
+
+### namespaceMap
+
+The namespaceMap uses the [term to IRI mapping](https://www.w3.org/TR/json-ld11/#example-11-term-expansion-from-context-definition) in the [JSON-LD context](https://www.w3.org/TR/json-ld11/#the-context).
+
+### element
+
+The [graph objects](https://www.w3.org/TR/json-ld11/#graph-objects) `@graph` lists the elements for the XCollection.
+
+The RDF graph of an instance of the SPDX model shall contain all Element nodes (i.e. objects that are subclasses of Element) as a list on top-level under the "@graph" key.
+This means that all references to Element nodes have to use the URI of the referenced Element.
+Inlining/Embedding of Element nodes into other nodes is not allowed.
+On the other hand, non-Element nodes (like those of type "ExternalReference" or similar complex data classes) have to be inlined (TODO: we may want to make an exception for CreationInfo, depending on the outcome of the surrounding discussion).
+
+## Context File
+
+The SPDX organization provides a context file that is to be used universally for all SPDX JSON-LD files of a given SPDX version.
```

Comment:
Suggested small tweak:

"The SPDX organization provides a global JSON-LD @context file that MUST be used universally for all SPDX JSON-LD files of a given SPDX version."

Reply:
Updated per suggestion